static/include/snippet/show-secret.md

```diff
@@ -0,0 +1,9 @@
+{{< alert title="Location secret" color="info" >}}
+By default, the sample code does not include your robot location secret.
+We strongly recommend that you add your location secret as an environment variable and import this variable into your development environment as needed.
+
+To show your robot's location secret in the sample code, toggle **Include secret** on the **Code sample** tab.
+You can also see your location secret on the [locations page](/manage/fleet/locations/).
```

Comment:
This may not work but worth trying
```suggestion
You can also see your location secret on the {{< glossary_tooltip term_id="location" text="location" >}}'s page.
```

Reply:
Hmm, not sure I see the value here: the user is not trying to better understand our fleet approach here, but trying to view and copy their secret. That secret lives on the locations page in the app, which I am trying to help them find by linking to that page's "reference page" in our docs (`/fleet/manage/`). While the tooltip clarification is helpful in general, I don't think it answers the most likely question here: "where can I get my secret?", which the linked-to page does attempt to do.

Happy to hear your reasoning!